src/main/java/lottoauto/domain/LottoStatus.java

```diff
@@ -0,0 +1,5 @@
+package lottoauto.domain;
+
+public enum LottoStatus {
```

Comment:
```suggestion
public enum LottoStatus(int price) {
```
enum 은 위와 같이 특정 값을 저장할 수 있습니다 ㅎㅎ